### ISSUE TITLE
(REQUEST) Only keep specific IV if greater than set CP

### ISSUE THREAD

**Issue by Pokemongobottttttt** [NECROBOTIO/NecroBot#3332]
I think it would be nice if it transferred those 10 CPs just because their 95 IV. I know they have potential but I am not about to dump 100k's worth of start dust into them. Instead of IV or CP, can we get a filter for both, 

Keep miniumum IV of 93 and up only if CP greater than 1000

**Comment by spamfighter82** [comment]
```
"KeepMinCp": 1250,
"KeepMinIvPercentage": 85.0,
"KeepMinOperator": "and",
```

or for individual pkmn

```
    "Arcanine": {
      "KeepMinCp": 1750,
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 90.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "and",
      "MovesOperator": "or"
    },
```

**Comment by jck357** [comment]
But maybe OP wants to keep all pokemon above cp 2000 (with bad iv) too for example.

**Comment by Pokemongobottttttt** [comment]
Yeah I would still like to keep anything over 1800 CP reguardless of IV. 

In addition that the original request, Keep IV 93 and up only if above 1000 CP

**Comment by Pokemongobottttttt** [comment]
Would this be easy to write in or would it take some testing?

**Comment by nfettinger** [comment]
The current per Pokemon configuration does not allow this currently?

**Comment by Pokemongobottttttt** [comment]
If i do this:
   "KeepMinCp": 1000,
   "KeepMinIvPercentage": 93.0,
   "KeepMinOperator": "and",

Plus config the per pokemon:
    "Arcanine": {
      "KeepMinCp": 1800
      "KeepMinLvl": 6,
      "UseKeepMinLvl": false,
      "KeepMinIvPercentage": 1.0,
      "KeepMinDuplicatePokemon": 1,
      "Moves": [],
      "KeepMinOperator": "and",
      "MovesOperator": "or"
    },

Wont this transfer any 1000 CP 93IV arcanine? Instead of keeping those as well
